### PR TITLE
Create Word Form Part of speech optional

### PR DESCRIPTION
**Pull request by gmcauliffe**
- Updated Create WOrd form so that part of speech defaults to "Please Select:" and is an optional entry.
- Added a "placeholder" boolean prop to "Directory list", if true will add a placeholder to the field.
- Updated field.js email verification to pass  Husky

### CODE REVIEW

File: frontend/app/assets/javascripts/views/pages/explore/dialect/create/Word/validation.js [hunk 1]
```diff
@@ -5,10 +5,7 @@ import copy from './internationalization'
 
 const validForm = yup.object().shape({
   'dc:title': yup.string().required(copy.validation.title),
-  'fv-word:part_of_speech': yup
-    .string()
-    .label('Part of speech')
-    .required(),
+  'fv-word:part_of_speech': yup.string().label('Part of speech'),
```
Comment (coakenfold) [inline frontend/app/assets/javascripts/views/pages/explore/dialect/create/Word/validation.js]: Could the string 'Part of speech' be added to the internationalization.js file @ frontend/app/assets/javascripts/views/pages/explore/dialect/create/Word/internationalization.js?

Like how line 7 is doing it above

Comment (gmcauliffe) [reply frontend/app/assets/javascripts/views/pages/explore/dialect/create/Word/validation.js]: Definitely, can do.
On review though, do we still need to use ".label" to override the key name if it is no longer a required field?

Comment (coakenfold) [reply frontend/app/assets/javascripts/views/pages/explore/dialect/create/Word/validation.js]: Yeah you're right, it's not needed on an optional field

**Comment by coakenfold** [comment]
Thanks!

**Pull request merged**